### PR TITLE
Minor spelling and formatting fixes

### PR DESCRIPTION
**Pull request by Flash1232**

### CODE REVIEW

File: app/src/main/java/org/grapheneos/apps/client/service/SeamlessUpdaterJob.kt [hunk 1]
```diff
@@ -74,7 +74,7 @@ class SeamlessUpdaterJob : JobService() {
 
                 if (requireConfirmation.isNotBlank() && result.requireConfirmation.isNotEmpty()) {
                     if (content.isNotBlank()) content += ", "
-                    content += "$requireConfirmation update update available."
```
Comment (thestinger) [inline app/src/main/java/org/grapheneos/apps/client/service/SeamlessUpdaterJob.kt:77]: Saying that an update is available may be a better way to phrase it. A problem with changing so much at once is that the whole pull request will be stalled or rejected based on any single minor thing that's not desired.

**Comment by Flash1232** [comment]
I'll try to split it up and re-open another PR, sorry!

**Comment by thestinger** [comment]
@Flash1232 Would appreciate if you can do that. It would also be good if you did `git pull --rebase upstream` after adding this repository as the upstream or alternatively do it with `git rebase` when you want to update it rather than using merge or whatever you used.

**Comment by thestinger** [comment]
It's just difficult to do something with a PR that changes a bunch of different things. If there's one little thing that isn't really wanted or where we're neutral about whether it should be changed then it ends up stalling everything else. It's much easier for me to merge 10 different tiny PRs than to deal with 1 PR doing 10 things. When possible, it's really nice for it to be split up.

No need to be sorry about it. It's probably not obvious that submitting a dozen tiny PRs is much nicer for us to handle than submitting one in any case where it's not simply an obvious objective fix. Subjective can get stalled without a clear resolution and it may take time to figure out what's wanted, etc.

**Comment by Flash1232** [comment]
Thanks for the heads up! I'll practice my rebase skills on GOS with PR's to come! :)
It makes total sense, but yeah not very obvious to make N PRs.

Thank you!

**Pull request closed without merging**